lib/stanford-corenlp-full-2015-12-09/patterns/stopwords.txt

```diff
@@ -0,0 +1,257 @@
+!!
```

Comment:
Are English stopwords relevent for German classification?  Seems unlikely.  Is there an equivalent for the German language?